<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addservice</class>
 <widget class="QDialog" name="addservice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_Sair">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cadastrar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_ShortDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>32</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="placeholderText">
    <string>Descrição curta do Serviço</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Emoticon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Feedback">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Todos os campos devem ser preenchidos.</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_FullDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>681</width>
     <height>351</height>
    </rect>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoNone</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Cidade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Custo do Serviço:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinCusto">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
